weasyl/middleware.py

```diff
@@ -132,30 +163,52 @@ def __call__(self, environ, start_response):
 
 class SentryEnvironmentMiddleware(object):
     def __init__(self, app, dsn, reactor=None):
-        import raven
         self.app = app
-        self.client = raven.Client(dsn)
+        self.client = raven.Client(
+            dsn=dsn,
+            release=raven.fetch_git_sha(os.path.abspath(os.path.join(__file__, '../..'))),
```

Comment:
Is a shorthash fine? I don't have a local sentry to test again. I don't even know where this gets used in its UI.